portfolio/src/main/webapp/script.js

```diff
@@ -1,28 +1,52 @@
 // Copyright 2019 Google LLC
 //
-// Licensed under the Apache License, Version 2.0 (the "License");
-// you may not use this file except in compliance with the License.
-// You may obtain a copy of the License at
+// Licensed under the Apache License, Version 2.0 (the "License"); you may not
+// use this file except in compliance with the License. You may obtain a copy of
+// the License at
 //
 //     https://www.apache.org/licenses/LICENSE-2.0
 //
 // Unless required by applicable law or agreed to in writing, software
-// distributed under the License is distributed on an "AS IS" BASIS,
-// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
-// See the License for the specific language governing permissions and
-// limitations under the License.
+// distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+// WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+// License for the specific language governing permissions and limitations under
+// the License.
+
+// random image generator
+window.onload = SetImageOnMouseOver;
 
 /**
- * Adds a random greeting to the page.
+ * Change the given image to a random one whenever the user hovers over
  */
-function addRandomGreeting() {
-  const greetings =
-      ['Hello world!', '¡Hola Mundo!', '你好，世界！', 'Bonjour le monde!'];
+function SetImageOnMouseOver() {
+  document.getElementById('rand-front-tile')
+      .addEventListener('mouseover', function() {
+        getRandomFile(
+            /* element= */ document.querySelector('#rand-back-tile img'),
+            /* path= */ 'images/random/', /* fileName= */ 'test',
+            /* numElems= */ 4)
+      });
+}
 
-  // Pick a random greeting.
-  const greeting = greetings[Math.floor(Math.random() * greetings.length)];
+/**
+ *
```

Comment:
Generally if a function has any comment, it should include the one-liner overview verb phrase of the style "Does the thing."

https://google.github.io/styleguide/jsguide.html#jsdoc-method-and-function-comments